components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -184,18 +181,12 @@ export default class PostList extends React.PureComponent {
     }
 
     componentDidUpdate(prevProps, prevState, snapshot) {
-        if (!this.postListRef.current || !snapshot) {
-            return;
-        }
-
-        const postlistScrollHeight = this.postListRef.current.scrollHeight;
-        const postsAddedAtTop = this.props.postListIds.length !== prevProps.postListIds.length && this.props.postListIds[0] === prevProps.postListIds[0];
-        const channelHeaderAdded = this.props.olderPosts.allLoaded !== prevProps.olderPosts.allLoaded;
-        if ((postsAddedAtTop || channelHeaderAdded) && !this.state.atBottom) {
-            const scrollValue = snapshot.previousScrollTop + (postlistScrollHeight - snapshot.previousScrollHeight);
-            if (scrollValue !== 0 && (scrollValue - snapshot.previousScrollTop) !== 0) {
-                //true as third param so chrome can use animationFrame when correcting scroll
-                this.listRef.current.scrollTo(scrollValue, scrollValue - snapshot.previousScrollTop, true);
+        if (this.postListRef.current && snapshot) {
+            const postlistScrollHeight = this.postListRef.current.scrollHeight;
+            const scrollValue = postlistScrollHeight - snapshot.previousScrollHeight;
+            if (scrollValue !== 0) {
+                //true as second param so chrome can use animationFrame when correcting scroll
+                this.listRef.current.scrollTo(scrollValue, true);
```

Comment:
I don't think this works unless we're changing something in our fork of react-window since that method takes 3 parameters. I honestly don't understand what that second parameter did even though even though I'm looking at the code.

Reply:
Yup, this wont work without change `react-window`. You can do `this.listRef.current.scrollTo(scrollValue, null, true);`  for now.

The second param was added for browsers which does not support `scrollBy` so they can use `scrollTo`

Reply:
https://github.com/mattermost/mattermost-webapp/pull/3436
Edge does not have element.scrollBy so we need the second param here https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollBy#Browser_compatibility